modules/xcode/xcode_common.lua

```diff
@@ -892,11 +892,15 @@
 			settings['EXECUTABLE_PREFIX'] = cfg.buildtarget.prefix
 		end
 
-		--[[if cfg.targetextension then
+		if cfg.kind ~= "ConsoleApp" and cfg.targetextension then
```

Comment:
Is there any particular reason why a `ConsoleApp` can't specify a `targetextension`? `app.out` for example?

Reply:
In xcode there is practically no way to add extensions at ConsoleApp.
Even if you put values in `WRAPPER_EXTENTION` or` EXECUTABLE_EXTENSION`, the product has no extension.

Reply:
Oh ok, interesting. Thanks for the clarification!